mod_deps.properties

```diff
@@ -1,8 +1,9 @@
 # mod_deps should be 'after:<mod identifier>' (unquoted, semicolon-separated, \escaped for newline, see example below)
-bopver=1.10.2-5.0.0.2067
-acver=1.10.2-1.9.2.5
-depstring=after:BiomesOPlenty@[5.0.0.2067,);\
-            after:AbyssalCraftAPI@[1.8,);\
+bopver=1.10.2-5.0.0.2085
+acver=1.10.2-1.9.2.9
+depstring=after:BiomesOPlenty@[5.0.0.2085,);\
+            after:AbyssalCraftAPI@[1.8.1,);\
             after:BiomesYouGo@[3.0.2,);\
-            after:flowercraft;\
-            after:sugiforest
+            after:flowercraft@[6.01,);\
```

Comment:
One of those mods wasn't setting the version in their mod container class, but in that case it will be set from the mcmod.info, which had a version set.
I never tested this, but I don't see any reason why there would be a problem here.